dvol_python/test_dvol.py

```diff
@@ -95,14 +112,18 @@ def test_create_volume(self):
     def test_create_volume_already_exists(self):
         dvol = VoluminousOptions()
         dvol.parseOptions(ARGS + ["-p", self.tmpdir.path, "init", "foo"])
+        expected_output = "Error: volume foo already exists"
         try:
             dvol.parseOptions(ARGS + ["-p", self.tmpdir.path, "init", "foo"])
             # TODO assert exit code != 0
             self.assertTrue(dvol.voluminous.getOutput()[-1].strip().endswith(
```

Comment:
This should really be assertEqual as you're comparing things. I understand this is an issue with the existing test so it's probably ok not to fix it now.